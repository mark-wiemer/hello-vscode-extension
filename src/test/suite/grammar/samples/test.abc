a
(
    b
)
x
(
    (
        c
        xyz
    )
)
(
a