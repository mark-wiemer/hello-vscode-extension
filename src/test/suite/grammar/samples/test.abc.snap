>a
#^ source.abc keyword.letter
>(
#^ source.abc expression.group punctuation.paren.open
>    b
#^^^^ source.abc expression.group
#    ^ source.abc expression.group keyword.letter
>)
#^ source.abc expression.group punctuation.paren.close
>x
#^ source.abc keyword.letter
>(
#^ source.abc expression.group punctuation.paren.open
>    (
#^^^^ source.abc expression.group
#    ^ source.abc expression.group expression.group punctuation.paren.open
>        c
#^^^^^^^^ source.abc expression.group expression.group
#        ^ source.abc expression.group expression.group keyword.letter
>        xyz
#^^^^^^^^ source.abc expression.group expression.group
#        ^ source.abc expression.group expression.group keyword.letter
#         ^ source.abc expression.group expression.group keyword.letter
#          ^ source.abc expression.group expression.group keyword.letter
>    )
#^^^^ source.abc expression.group expression.group
#    ^ source.abc expression.group expression.group punctuation.paren.close
>    Q
#^^^^^^ source.abc expression.group
>)
#^ source.abc expression.group punctuation.paren.close
>(
#^ source.abc expression.group punctuation.paren.open
>a
#^ source.abc expression.group keyword.letter